src/view/pane/chat.js

```diff
@@ -293,9 +293,18 @@ Candy.View.Pane = (function(self, $) {
           e.stopPropagation();
         });
         $('#chat-autoscroll-control').click(self.Chat.Toolbar.onAutoscrollControlClick);
-
-        var a = document.createElement('audio');
-        self.Chat.Toolbar._supportsNativeAudio = !!(a.canPlayType && a.canPlayType('audio/mpeg;').replace(/no/, ''));
+		if( !!document.createElement('audio').canPlayType ) {
+			var a = document.createElement('audio');
+			if( !!(a.canPlayType('audio/mpeg;').replace(/no/, '')) ) {
```

Comment:
This block of if/else tests needs to be wrapped in a try catch for IE9. (See #357 & #356)